frontend/src/renderer/pages/Process.svelte

```diff
@@ -245,17 +245,7 @@
   {/if}
   <Layout {tabs} bind:tabList />
   <botnav class="dark:bg-transparent">
-    {#if tabs[$processState.currentTab + 1]?.name === "Advanced Options"}
-      <button on:click={nextTab}>Optional Filtering</button>
-      <button
-        on:click={() => {
-          if ($processState.completedTabs[$processState.currentTab]) {
-            $processState.currentTab += 2;
-          }
-        }}
-        class="nextBtn">Next: Skip Optional Filtering</button
-      >
-    {:else if tabs[$processState.currentTab + 1]?.name === "Processing"}
+    {#if tabs[$processState.currentTab + 1]?.name === "Processing"}
```

Comment:
Moved this to the sorter page so there can actually be page-specific validation. Ideally we'll move these all out of here because THEY DONT BELONG HERE